app/models/student.rb

```diff
@@ -32,10 +32,16 @@ class Student < ApplicationRecord
   validates :osis, length: { is: 9 },
                    numericality: { greater_than: 0,
                                    message: 'cannot be negative' }
+  validates :gender, inclusion: { in: %w(Female Male Other) }
```

Comment:
Not that it matters, but it doesn't have to be capitalized as a value in the db, but possibly in presentation.

So allowed values could just be `%w(female male other)`. Side effect would be it's easy to compare with `String.downcase`.

Reply:
can i just keep it as such

Reply:
ya